.github/workflows/pypi-tests.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       run: |
         pip install pip setuptools wheel -U
         pip install dask[complete]
+        pip install git+https://github.com/scikit-hep/awkward@main
         pip install --ignore-installed -q --no-cache-dir -e .[complete]
```

Comment:
```suggestion
        pip install git+https://github.com/scikit-hep/awkward@main
        pip install -q --no-cache-dir -e .[complete]
```

Reply:
Thanks a lot! surprised this wasn't an issue earlier... what I get for not carefully adjusting CI configs when taking inspiration from another one of my projects  🤦